dashboard/app/lib/javascripts/dashboard/views/apps-list.js.jsx

```diff
@@ -17,8 +17,16 @@ var AppsList = React.createClass({
 		return (
 			<ul className="items-list">
 				{apps.map(function (app) {
+					var classes = [];
+					if (assertEqual(app.id, selectedAppId)) {
+						classes.push("selected");
+					}
+					if (typeof app.meta["flynn-system-app"] !== undefined &&
```

Comment:
No need to test for `undefined`, the latter assertion alone is fine. (Also `app.meta.hasOwnProperty('flynn-system-app')` is more idiomatic.)